src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -221,6 +222,12 @@ The combination of HTTP/2 and HTTP/2 server push provides the _benefits_ of
 bundling (reduced latency) without actual bundling. Keeping resources separate
 means they can be cached efficiently and be shared between pages.
 
+HTTP/2 Push needs to be utilized with care, as it forces data to the browser,
```

Comment:
Taking a closer look, I guess this edit is fine given the current form of the doc, but I'd like to work on getting the whole H/2 background section lifted out and put somewhere else...